tests/test_climsoft.py

```diff
@@ -0,0 +1,100 @@
+import random
+
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text as sa_text
+from sqlalchemy.orm import sessionmaker
+from opencdms.models.climsoft import v4_1_1_core as climsoft
+
+DB_URL = 'mysql+mysqldb://root:password@127.0.0.1:33308/mariadb_climsoft_db_v4'
+
+db_engine = create_engine(DB_URL)
+
+station_data = dict(
+    stationId=str(random.randint(1000, 2000)),
+    stationName='Test Station',
+    country='UK'
+)
+
+
+@pytest.fixture
+def db_session():
+    Session = sessionmaker(bind=db_engine)
+    session = Session()
+    yield session
+    session.close()
+
+
+def setup_module(module):
+    with db_engine.connect() as connection:
+        trans = connection.begin()
+        connection.execute('SET FOREIGN_KEY_CHECKS = 0;')
```

Comment:
as I was deleting all the tables in the database before each test, I wrote it like this. Since Climsoft doesn't implement cascade, we don't have to set foreign checks to zero.

Reply:
Let's keep the disabling of foreign key checks - it could be important to consider later